### PR TITLE
Add tool set translations

### PR DESCRIPTION
**Pull request by eSlider**
* [x] Added new translations for three languages I can (Please review!)
* [x] Registered new tool set translations
* [x] Rewrite default subjects if translations available

### CODE REVIEW

File: Resources/public/mapbender.element.digitizer.js [hunk 1]
```diff
@@ -911,7 +904,7 @@
 
                 schema.schemaName = schemaName;
 
-                var toolset = widget.toolsets[schema.featureType.geomType];
+                var toolset = widget.toolsets[schema.featureType.geomType ];
```
Comment (atd-schubert) [inline Resources/public/mapbender.element.digitizer.js]: unnecessary change!

Comment (eSlider) [reply Resources/public/mapbender.element.digitizer.js]: @atd-schubert, :)
